tests/gui/conversation/export/test_device.py

```diff
@@ -222,7 +222,7 @@ def test_Device_print_file_when_orig_file_already_exists(homedir, config, mocker
 
 def test_Device_print_file_when_orig_file_already_exists_not_qubes(homedir, config, mocker, source):
     """
-    The signal `print_requested` should still be emmited if the original file already exists.
+    The signal `print_requested` should not be emited if the original file already exists.
     """
```

Comment:
Admittedly, I'm not very familiar with the printer workflow. Was this a bug in our test case or did the print behaviour change? (Would you mind elaborating just a little on the scenario that the test case description refers to?)

Reply:
Well, the test name indicated that what was being tested was the printing feature, but the `export_file_to_usb_drive` method was being tested, instead of `print_file`. The surroundings make me think this was a copy-paste mistake.

Besides that, the assertion `assert len(print_requested_emissions) == 0` (that was not changed) does assert that the signal is **not** emitted, which matches the code behavior so I fixed the copy-pasted text description.

Eventually, these tests would gain in being reconsidered, but I'd rather not doing that while also modifying the production code. Even if some of them bring relatively little guarantees that the production code is correct, they're the current baseline and I'm keen on treating them as such while performing these first round of refactoring. As we progress separating concerns and separating queries from commands,  I am confident testing behavior will become easier and we'll be able to write tests that are overall simpler, and make harder to miss mistakes like this one.

Reply:
Thank you for the clarification, much appreciated